samples/video/capture_and_display/README.rst

```diff
@@ -0,0 +1,60 @@
+.. _video_capture_display-sample:
+
+Video Capture And Display
+#########################
+
+Description
+***********
+
+This sample application uses the Video API to retrieve video frames from the
+video capture device, and writes frames to a video display device.
+
+Requirements
+************
+
+This sample requires a video capture device (e.g. a camera).
```

Comment:
... and a display.

(The mimxrt1064_evk board has a display connector, but no built-in display.)

Reply:
fixed